data/xml/W19.xml

```diff
@@ -13922,4 +13922,460 @@ In this tutorial on MT and post-editing we would like to continue sharing the la
+      <author><first>Ophélie</first><last>Lacroix</last></author>
+      <pages>136–143</pages>
+      <url>W19-7716</url>
+    </paper>
+    <paper id="17">
+      <title>Towards Deep Universal Dependencies</title>
+      <author><first>Kira</first><last>Droganova</last></author>
+      <author><first>Daniel</first><last>Zeman</last></author>
+      <pages>144–152</pages>
+      <url>W19-7717</url>
+    </paper>
+    <paper id="18">
+      <title>Delimiting Adverbial Meanings. A corpus-based comparative study on <fixed-case>C</fixed-case>zech spatial prepositions and their <fixed-case>E</fixed-case>nglish equivalents</title>
+      <author><first>Marie</first><last>Mikulová</last></author>
+      <author><first>Veronika</first><last>Kolářová</last></author>
+      <author><first>Jarmila</first><last>Panevová</last></author>
+      <author><first>Eva</first><last>Hajičová</last></author>
+      <pages>153–159</pages>
+      <url>W19-7718</url>
+    </paper>
+    <paper id="19">
+      <title>A <fixed-case>S</fixed-case>panish E-dictionary of Collocations</title>
+      <author><first>Maria Auxiliadora Barrios</first><last>Rodriguez</last></author>
+      <author><first>Igor</first><last>Boguslavsky</last></author>
+      <pages>160–167</pages>
+      <url>W19-7719</url>
+    </paper>
+    <paper id="20">
+      <title>Exceptive constructions. A Dependency-based Analysis</title>
+      <author><first>Mohamed</first><last>Galal</last></author>
+      <author><first>Sylvain</first><last>Kahane</last></author>
+      <author><first>Yomna</first><last>Safwat</last></author>
+      <pages>168–174</pages>
+      <url>W19-7720</url>
+    </paper>
+    <paper id="21">
+      <title>Quantitative Analysis on verb valence evolution of <fixed-case>C</fixed-case>hinese</title>
+      <author><first>Bingli</first><last>Liu</last></author>
+      <author><first>Chunshan</first><last>Xu</last></author>
+      <pages>175–180</pages>
+      <url>W19-7721</url>
+    </paper>
+    <paper id="22">
+      <title>Association Metrics in Neural Transition-Based Dependency Parsing</title>
+      <author><first>Patricia</first><last>Fischer</last></author>
+      <author><first>Sebastian</first><last>Pütz</last></author>
+      <author><first>Daniël</first><last>de Kok</last></author>
+      <pages>181–189</pages>
+      <url>W19-7722</url>
+    </paper>
+    <paper id="23">
+      <title>Presenting <fixed-case>TWITTIRÒ</fixed-case>-<fixed-case>UD</fixed-case>: An <fixed-case>I</fixed-case>talian Twitter Treebank in Universal Dependencies</title>
+      <author><first>Alessandra Teresa</first><last>Cignarella</last></author>
+      <author><first>Cristina</first><last>Bosco</last></author>
+      <author><first>Paolo</first><last>Rosso</last></author>
+      <pages>190–197</pages>
+      <url>W19-7723</url>
+    </paper>
+    <paper id="24">
+      <title>Pān.inian Syntactico-Semantic Relation Labels</title>
```

Comment:
Should be Pāṇinian (note the n with the dot beneath).

Reply:
Good eye, thank you!